go/vt/vtadmin/cluster/discovery/discovery.go

```diff
@@ -75,4 +75,5 @@ func New(impl string, cluster string, args []string) (Discovery, error) {
 
 func init() { // nolint:gochecknoinits
 	Register("consul", NewConsul)
+	Register("static", NewStaticFile)
```

Comment:
Oh, I wonder ... what do you think about "static" vs "staticfile"?

Reply:
i like

Reply:
err 😅 I meant the other way haha

because it's both static and sourced from a file. but i don't actually feel that strongly

Reply:
Ah, sorry for misunderstanding. Your request was so polite. :')